app/components/content/edit-form.js

```diff
@@ -384,7 +384,13 @@ const EditForm = ({
         <NewSelect
           className='basic-single'
           classNamePrefix='select'
-          defaultValue={options[0]}
+          defaultValue={
+            !subtype
+              ? options[0]
+              : options.filter(x => {
+                if (x.value === subtype) return x
+              })[0]
```

Comment:
```suggestion
              : options.find(x => x.value === subtype)
```

Simpler using `Array#find`

Reply:
Thank you for sharing your knowledge 😊 (helps me learn these methods too)